### PR TITLE
Replace ATs by ITs

### LINKED ISSUE #333: Refactor the ATs to allow running the the SCDF ITs.
- Replace the outdated ATs tests with test sets form the SCDF ITs test jar.
- Extend the `create` scripts to allow pre-registering of bulk stream and task apps. Provide reasonable defaults for each platform. 

Depends on https://github.com/spring-cloud/spring-cloud-dataflow/pull/4401

### PR DESCRIPTION
**Pull request by tzolov**
- Remove existing ATs tests, test utilities/DSLs.
 - Add an empty DataFlowAT test class to extend all DataFlowIT tests.
 - Add an empty TaskScheduleAT test class to extend all TaskScheduleIT tests.
 - Add create#apps phase to the scripts to allow pre-registering the bulk stream and task apps.
 - Add -sa and -ta parameter to the create script to allow passing the stream and task bulk apps uris.

 depends on https://github.com/spring-cloud/spring-cloud-dataflow/pull/4401

 Resolves #333

### CODE REVIEW

File: kubernetes/skipper-server/server-uri.sh [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
   SKIPPER_IP=$(kubectl get svc --namespace $KUBERNETES_NAMESPACE | grep skipper | awk '{print $4}')
-  if [[ "$SKIPPER_IP" =~ [\d{1-3}:\d{1-3}:\d{1-3}:\d{1-3}] ]] ;
-  then
-       export SKIPPER_SERVER_URI="http://$SKIPPER_IP"
-       wait_for_200 ${SKIPPER_SERVER_URI}/api/about
-  else
-      echo "Unable to determine skipper server host $SKIPPER_IP Did deployment fail?"
-      exit 1
-  fi
+  while [[ "$SKIPPER_IP" == "<pending>" ]] ; do
```
Comment (dturanski) [inline kubernetes/skipper-server/server-uri.sh:4]: Why did you change this?

Comment (tzolov) [reply kubernetes/skipper-server/server-uri.sh:4]: We changed this together with @chrisjs because the old conditions were bogus and were letting the `SKIPPER_SERVER_URI` be set to `http:<pending>`. That in turn meant that the `wait_for_200 ${SKIPPER_SERVER_URI}/api/about` check fails.
The change above waits until the LoadBalancer for the Skipper is set and valid external IP is produced. 
Same change is applied to the SCDF script as well.

Comment (chrisjs) [reply kubernetes/skipper-server/server-uri.sh:4]: side note; the external ip field may not always be an IP but rather a hostname on say AWS

**Comment by jvalkeal** [comment]
merged per 23978b12f405624a7953a150f26afd10c3005c61
